From e6e26e488d78d4b1f86aa50f78c7f49d450f15b6 Mon Sep 17 00:00:00 2001
Message-Id: <e6e26e488d78d4b1f86aa50f78c7f49d450f15b6.1711648979.git.keller@fs-net.de>
In-Reply-To: <bb92f3f011b7b125a61ac948b576f518cee03bb3.1711648979.git.keller@fs-net.de>
References: <bb92f3f011b7b125a61ac948b576f518cee03bb3.1711648979.git.keller@fs-net.de>
From: Hong Shi <hongsh@codeaurora.org>
Date: Thu, 16 Jul 2020 16:54:35 +0800
Subject: [PATCH 08/16] qcacld-2.0: iapp_work.deferred_work used but not
 initialized

iapp_work.deferred_work used but not initialized. Ailgn the compile
condition between use and init.

Change-Id: I1f3c95764e6e118a88a686257e8ee38220c3e03c
CRs-Fixed: 2731922
---
 CORE/CLD_TXRX/TLSHIM/tl_shim.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CORE/CLD_TXRX/TLSHIM/tl_shim.c b/CORE/CLD_TXRX/TLSHIM/tl_shim.c
index 7f44e92..d90dec4 100644
--- a/CORE/CLD_TXRX/TLSHIM/tl_shim.c
+++ b/CORE/CLD_TXRX/TLSHIM/tl_shim.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2013-2019 The Linux Foundation. All rights reserved.
+ * Copyright (c) 2013-2020 The Linux Foundation. All rights reserved.
  *
  * Previously licensed under the ISC license by Qualcomm Atheros, Inc.
  *
@@ -2090,7 +2090,7 @@ VOS_STATUS WLANTL_Close(void *vos_ctx)
 #endif
 
 	adf_os_mem_free(tl_shim->vdev_active);
-#ifdef FEATURE_WLAN_ESE
+#if defined(FEATURE_WLAN_ESE) && !defined(FEATURE_WLAN_ESE_UPLOAD)
 	vos_flush_work(&tl_shim->iapp_work.deferred_work);
 #endif
 	vos_flush_work(&tl_shim->cache_flush_work);
-- 
2.14.5

